pkg/services/mssqldr/catalog.go

```diff
@@ -0,0 +1,58 @@
+package mssqldr
+
+import "github.com/Azure/open-service-broker-azure/pkg/service"
+
+// nolint: lll
+func (m *module) GetCatalog() (service.Catalog, error) {
+
+	return service.NewCatalog([]service.Service{
+		// dbms pair registered service
+		service.NewService(
+			service.ServiceProperties{
+				ID:             "00ce53a3-d6c3-4c24-8cb2-3f48d3b161d8",
+				Name:           "azure-sql-12-0-dr-dbms-pair-registered",
+				Description:    "Azure SQL 12.0-- disaster recovery DBMS pair registered",
+				ChildServiceID: "2eb94a7e-5a7c-46f9-b9d2-ff769f215845", // More children in fact
+				Metadata: service.ServiceMetadata{
+					DisplayName:      "Azure SQL 12.0-- disaster recovery DBMS Pair registered",
+					ImageURL:         "https://azure.microsoft.com/svghandler/sql-database/?width=200",
+					LongDescription:  "Azure SQL 12.0-- disaster recovery DBMS pair registered, as the primary server and the secondary server of failover groups",
+					DocumentationURL: "https://docs.microsoft.com/en-us/azure/sql-database/",
+					SupportURL:       "https://azure.microsoft.com/en-us/support/",
+				},
+				Bindable: false,
+				Tags: []string{
+					"AzurePublicCloud",
```

Comment:
I think tags `Azure*Cloud` should be removed.

Reply:
Yeah, thanks for the good catch.